slasher/detection/testing/utils.go

```diff
@@ -0,0 +1,39 @@
+package testing
+
+import (
+	"crypto/rand"
+
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/shared/params"
+)
+
+// SignedBlockHeader given slot, proposer index this function generates signed block header.
+// with random bytes as its signature.
+func SignedBlockHeader(slot uint64, proposerIdx uint64) (*ethpb.SignedBeaconBlockHeader, error) {
+	sig, err := genRandomSig()
+	if err != nil {
+		return nil, err
+	}
+	root := [32]byte{1, 2, 3}
+	return &ethpb.SignedBeaconBlockHeader{
+		Header: &ethpb.BeaconBlockHeader{
+			ProposerIndex: proposerIdx,
+			Slot:          slot,
+			ParentRoot:    root[:],
+			StateRoot:     root[:],
+			BodyRoot:      root[:],
+		},
+		Signature: sig,
+	}, nil
+}
+
+func genRandomSig() (blk []byte, err error) {
+	blk = make([]byte, 96)
+	_, err = rand.Read(blk)
+	return
```

Comment:
Is something supposed to be returned here?

Reply:
good catch. thanks